libnmstate/nm/wired.py

```diff
@@ -84,6 +89,11 @@ def get_info(device):
     except AttributeError:
         pass
 
+    mac = device.get_hw_address()
+    # A device may not have a MAC or it may not yet be "realized" (zeroed mac).
+    if mac and mac != '00:00:00:00:00:00':
```

Comment:
not a blocker, should the zero mac be a constant?

Reply:
Done